isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

```diff
@@ -281,6 +281,46 @@ OpenID Connect Abfragen zur Identität eines Benutzers, OpenID-Token oder Custom
 [[authentifizierung-und-autorisierung]]
 === Authentifizierung & Autorisierung
 
+Gemäß den Vorgaben des IdentityAccessManagements (IAM), das in der  xref:isyfact-standards-doku:referenzarchitektur:fachlich.adoc#[IsyFact-Referenzarchitektur] zum Einsatz kommt, wird die Authentifizierung & Autorisierung über Multi-Realm bzw. Multi-Tenancy organisiert. Nach den vordefinierten xref:isyfact-standards-doku:referenzarchitektur:software-technisch.adoc#prinzipien-der-sicherheitsarchitektur[Prinzipien der Sicherheitsarchitektur] ergeben sich unterschiedliche Zugriffswege, wie das nachfolgende Bild zeigt:
+
+.Authentifizierungswege als Mandanten
+[id="image-multi-tenancy-multi-iam",reftext="{figure-caption} {counter:figures}"]
+image::konzept/security_multi-tenancy_with_multi_iams.dn.svg[align="center"]
```

Comment:
Ich würde in der Beschreibung noch plakativ etwas Werbung machen, dass unser Baustein Multi-Tenancy unterstützt. Damit sollte es doch auch möglich sein, dass ich unterschiedliche IAM-Server nutze, da ich ja für jeden Tenant die URI gesondert angebe.

Reply:
Ja genau das war nicht nur im Text, sondern auch schon in der Grafik enthalten. Habe es wieder aufgenommen.